synapse/federation/transport/client.py

```diff
@@ -407,12 +407,12 @@ def query_client_keys(self, destination, query_content, timeout):
                 "<user_id>": {
                   "<device_id>": {...}
               } }
```

Comment:
could you structure this to make it more legible, and also include the commas to make it valid json?

```suggestion
                }
              },
```